packages/acs-ui-common/package.json

```diff
@@ -6,25 +6,24 @@
   "main": "dist/dist-cjs/index.js",
   "types": "dist/acs-ui-common.d.ts",
   "scripts": {
-    "build": "rushx clean && rushx build:esm && rushx build:cjs && api-extractor run --local",
-    "build:esm": "tsc",
+    "build": "rushx by-flavor \"npm run build:by-flavor\"",
+    "build:esm": "if-env FLAVOR=stable && rushx preprocess && tsc -project tsconfig.preprocess.json || (if-env FLAVOR=beta && tsc)",
     "build:cjs": "rollup -c --silent --failAfterWarnings",
     "build:watch": "rushx build",
-    "clean": "rimraf dist",
-    "copy-original": "cpx ./src/** ./preprocessed",
-    "preprocess": "rushx copy-original && babel ./src --out-dir ../preprocessed --extensions \".ts,.tsx\" --keep-file-extension --config-file ./.babelrc.js --relative && rimraf ../preprocessed",
-    "preprocess:stable": "cross-env FLAVOR=stable rushx preprocess",
-    "clean:preprocess": "rimraf ./preprocessed",
-    "build:stable": "rushx clean:preprocess && rushx preprocess:stable && tsc -project tsconfig.preprocess.json && rushx build:cjs && api-extractor run --local",
-    "test:stable": "rushx clean:preprocess && rushx preprocess:stable && jest --verbose ./preprocessed -c jest.config.preprocess.js",
+    "clean": "rimraf dist && rimraf preprocessed",
+    "current-flavor": "echo You are running under FLAVOR: && env-cmd -f ../../common/config/env/.env node -p process.env.FLAVOR",
+    "by-flavor": "rushx current-flavor && env-cmd -f ../../common/config/env/.env --use-shell",
```

Comment:
All the `by-flavor` variants are not intended to be run directly. Could we

* Add a prefix to all such commands, perhaps `plumbing-` or `internal-`
* Move all of them to the bottom of the `scripts`. Perhaps prefix with `zzPlumbing-`, so that sorting will naturally drop them to the bottom of the list.

Reply:
To make it short, I've changed the name with prefix "_" (how we name internal functions and variables) and they've been moved to the bottom ;D